apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/DocumentDetailActivity.java

```diff
@@ -50,10 +57,22 @@
 
     private String mFullErrorContents;
 
+    private String mDocumentContent;
+
+    private String mDocumentDate;
+
+    private boolean mDocumentState;
+
+    private String mDocumentError;
+
+    private boolean mDocumentNullStatus;
+
     private ProgressBar mDetailProgress;
 
     private ListView mListView;
 
+    private MenuItem mRefreshDocument;
+
     private AppCenterConsumer<DocumentWrapper<TestDocument>> getAppDocument = new AppCenterConsumer<DocumentWrapper<TestDocument>>() {
```

Comment:
inconsistency with types, you changed code to always list as Map, but read as TestDocument, this will change the way we display properties on refresh.